forest/main.py

```diff
@@ -92,15 +92,18 @@ def main(argv=None):
         }
         dataset = drivers.get_dataset(group.file_type, settings)
         datasets[group.pattern] = dataset
-        viewer = dataset.map_view()
-        viewers[group.label] = viewer
-        renderers[group.label] = [
-                viewer.add_figure(f)
-                for f in figures]
+
+        # Add optional map view
+        if hasattr(dataset, "map_view"):
+            map_view = dataset.map_view()
+            map_views[group.label] = map_view
+            renderers[group.label] = [
+                    map_view.add_figure(f)
```

Comment:
Maybe not for this one, but perhaps `map_view.add_figure` should be called `map_view.add_renderer`?